app/components/request-archive/component.js

```diff
@@ -0,0 +1,10 @@
+import Component from '@glimmer/component';
+
+export default class RequestArchiveComponent extends Component {
+  get AllRequests() {
```

Comment:
consider to change `allRequests` - variable start with uppercase says us it should be a `class`